### PR TITLE
Add link to training materials

### PR DESCRIPTION
**Pull request by Robinlovelace**

### CODE REVIEW

File: regions_www/tabs/manual_body.html [hunk 1]
```diff
@@ -23,6 +23,10 @@ <h4><a name="essentials"></a>PCT Essentials</h4>
           <ul style="line-height:2.2;">
           <li>Read our two pager <a target="_blank" href="https://cdn.rawgit.com/npct/pct-shiny/30b98621b37da3cb66fcf2253583af9f026f9fce/regions_www/www/static/03d_other_reports/2020-pct-essentials.pdf">Essentials</a> document first, for a high level summary</li>
           </ul>
+          <h4><a name="advanced-training"></a>Advanced training guide (for developers / R users)</h4>
```
Comment (nikolai-b) [inline regions_www/tabs/manual_body.html:26]: This is missing the link on top

```html
<div style="padding-bottom:10px;"><a class="manuallinks" href="#advanced-training">Advanced training guide</a></div>
 ```

Comment (Robinlovelace) [reply regions_www/tabs/manual_body.html:26]: Thanks Nikolai. Updated as suggested, although not 100% sure this is now right.

**Comment by Robinlovelace** [comment]
Also this PR should be to master now.

**Comment by Robinlovelace** [comment]
> Why have you got a PR to merge into https://github.com/npct/pct-shiny/tree/pct_essentials branch? 

I don't anymore. I created it before the essentials branch was merged and left it open deliberately to make the useful conversations there more prominent. I should have said 'Replaces #831 ' in the discussion of #835 . Will do the that with this one.

**Pull request closed without merging**